tutorials/classical_simulation/Simulating with SPAM errors.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
<p>I'd rather use noiseless/perfect or "We compare simulations with and without noise" instead of clean</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>